.travis.yml

```diff
@@ -41,6 +41,7 @@ matrix:
         - cd dex-contracts && truffle compile && cd -
         - docker-compose up -d ganache-cli driver listener | while read line ; do echo "$(date)| $line"; done; # prints timestamps for profiling
       script:
+        - set -e # make this script fail as soon as any individual command fail
```

Comment:
Can you test this by changing the expected hash in one of the first tests to some wrong value and check if travis stops after that test instead of running through all test cases.

Reply:
Sure thing! Just a moment

Reply:
[Here](https://github.com/gnosis/dex-services/pull/163/commits/07fb46a242d6c7b21d39bd8b18de892e57c89d36) we are.

Reply:
Build failed as expected. [Check it out](https://travis-ci.org/gnosis/dex-services/builds/554130831?utm_source=github_status&utm_medium=notification)